content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -0,0 +1,89 @@
+==========
+AvaTax use
+==========
+
+AvaTax is a tax calculation software that can be integrated with Odoo in the United States and
+Canada. Once the :doc:`integration setup <../avatax>` is complete, the calculated tax is simple and
+automatic. This document covers the use of AvaTax in Odoo.
+
+Tax calculation
+===============
+
+Automatically calculate taxes on Odoo quotations and invoices with Avatax by confirming the
+documents. Alternatively, calculate the taxes manually by clicking the :guilabel:`Compute taxes
+using Avatax` button while these documents are in draft mode.
+
+.. tip::
+   Clicking the :guilabel:`Compute taxes using Avatax`button recalculates taxes if any product lines
+   are edited on the invoice.
+
+.. image:: avatax_use/calculate-avatax.png
+   :align: center
+   :alt: Sales quotation with the confirm and compute taxes using AvaTax buttons highlighted.
+
+The tax calculation is automatically triggered during the following circumstances:
+
+Auto Triggers
+-------------
+
+- When the sales rep sends the quote by email with :guilabel:`Send by email` button (pop-up).
+- When the customer views the online quote on the portal.
+- When a quote is confirmed and becomes a sales order.
+- When the customer views the invoice on the portal.
+- When a draft invoice is validated.
+- When the customer views the subscription in the portal.
+- When a subscription generates an invoice.
+- When the customer gets to the last screen of the eCommerce checkout.
```

Comment:
Do you think the list is necessary? I would generalize explaining that one tax by default is added on invoices and sales orders. If we list all cases, we might miss some, and I think it would be enough to have a couple of sentences saying that, by default, there is one. Then explain a bit more of the manual trigger.

Reply:
I agree with @chiaraprattico here, that unless we have a complete exhaustive list of all possible automatic triggers it is probably best to simplify to a general statement. If this is a complete list then we should be good to list as is 👍